src/main/java/com/bouncestorage/swiftproxy/v1/ContainerResource.java

```diff
@@ -237,6 +240,12 @@ public Response listContainer(@NotNull @PathParam("container") String container,
     @XmlRootElement(name = "object")
     @XmlType
     static class ObjectEntry {
+        static class SwiftDateSerializer extends DateSerializer {
```

Comment:
Ah, sorry. I was looking at the headers (although that one is actually in Unix epoch time format) -- never mind!